frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/flavor.tsx

```diff
@@ -127,7 +130,7 @@ export const FlavorSelect: React.FC<FlavorProps> = React.memo(
                     {isDefault
                       ? t('kubevirt-plugin~{{flavor}} (default)', { flavor: t(f.toString()) })
                       : t(f.toString())}
-                    {flavorDesc && f !== Flavor.CUSTOM && ` - ${flavorDesc}`}
+                    {flavorDesc && f !== Flavor.CUSTOM && `: ${flavorDesc}`}
```

Comment:
why we need to print flavorDesc twice, we already print it in L131 ?

Reply:
L131 only prints the name, not the CPU and memory piece. The reason this logic is so confusing is that it's trying to handle adding `(default)` to the default case AND only adding the memory and CPU piece to flavors that aren't `Custom`.